doc-content/kogito-docs/src/main/asciidoc/configuration/proc_messaging-enabling.adoc

```diff
@@ -0,0 +1,153 @@
+[id='proc_messaging-enabling_{context}']
+
+= Enabling messaging for {PRODUCT} services
+
+{PRODUCT} supports the MicroProfile Reactive Messaging specification for messaging in your services. You can enable messaging to configure messages as either input or output of business process execution.
+
+For example, the following process uses messaging start and end events to communicate with travelers:
+
+.Example process with messaging start and end events
+image::kogito/bpmn/bpmn-messaging-example.png[Image of message-based process]
+
+In this example, the message start and end events require the following information:
+
+* Message name that maps to the channel that delivers messages
+* Message payload that maps to a process instance variable
+
+.Example message configuration for start event
+image::kogito/bpmn/bpmn-messaging-start-event.png[Image of message start event data]
+
+.Example message configuration for end event
+image::kogito/bpmn/bpmn-messaging-end-event.png[Image of message end event data]
+
+{PRODUCT} messaging is based on https://kafka.apache.org/[Apache Kafka], so you must have Kafka installed in order to enable messaging. Your marshalling configuration depends on the messaging solution you use.
```

Comment:
Unclear on this because the following was the desired phrasing by Duncan and Ricardo in the Kogito on OCP doc:

```
{PRODUCT} supports the MicroProfile Reactive Messaging specification for messaging in your services. {PRODUCT} messaging is based on https://kafka.apache.org/[Apache Kafka], so you must have Kafka installed in order to enable messaging.
```

I'll ping the three of you in gchat about this. Thanks.

Reply:
Okay, confirmed the following wording with Ricardo in chat, with you pinged:

For this doc:

```
{PRODUCT} supports the MicroProfile Reactive Messaging specification for messaging in your services. ... For this procedure, the messaging is based on https://kafka.apache.org/[Apache Kafka], so you must have Kafka installed in order to enable messaging. Your marshalling configuration depends on the messaging solution you use.
```

For the OCP content, where Operator messaging is indeed based on Kafka:

```
{PRODUCT} supports the MicroProfile Reactive Messaging specification for messaging in your services. {PRODUCT} Operator messaging is based on https://kafka.apache.org/[Apache Kafka] and enables you to configure messages as either input or output of business process execution.
```

Reply:
looks good, thanks.